modules/fedimint-ln-client/src/endpoint_constants.rs

```diff
@@ -0,0 +1,2 @@
+pub const GET_GATEWAY_ID_ENDPOINT: &str = "id";
+pub const PAY_INVOICE_ENDPOINT: &str = "pay_invoice";
```

Comment:
If this was in `fedimint-ln-common` it would likely be more useful because we could use it for the API client too.

Reply:
Going to echo what Eric said here, we should not have to re-define `GET_GATEWAY_ID_ENDPOINT`.  Your `endpoint_constants.rs` file inside the gateway should go into `fedimint-ln-common` so it can be referenced by both the gateway and the client.

Reply:
Done.